cli/cfncluster/cfncluster.py

```diff
@@ -215,24 +215,25 @@ def stop(args):
     set_asg_limits(asg=asg, min=0, max=0, desired=0)
 
     # Stop master Server
-    master_server_id = get_master_server_id(stack_name, config)
-    ec2conn = boto.ec2.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
-                                                    aws_secret_access_key=config.aws_secret_access_key)
-    try:
-        response = ec2conn.stop_instances(master_server_id)
-    except boto.exception.BotoServerError as e:
-        if e.message.endswith("does not exist"):
-            print(e.message)
-            sys.stdout.flush()
+    if not args.compute_only:
```

Comment:
Suggestion:

if args.compute_only:
     return

 will improve the readability